translations/de/info/task_description.html

```diff
@@ -128,12 +125,12 @@
 </div>
 
 <p class="for_info_only">
-    <strong>How it is used: </strong>
-    This task is like pathfinding for numbers.
-    It shows how many things in this world can be represented with mathematics, even words.
+    <strong>Wozu das gut ist:</strong><br>
+    Diese Aufgabe ist eine Variante des <a href="https://de.wikipedia.org/wiki/Pathfinding">Pathfinding<a> für Zahlen.
```

Comment:
Not all the numbers in the given list are part of the correct chain: 
**Input:** [123, 991, 323, 321, 329, 121, 921, 125, 999] 
**Result:** [123, 121, 921, 991, 999]  

Actually, there are no words which are represented as numbers in this task, it only replaced words by numbers. But these numbers are not representing words... So I removed that sentence as it seems to be a bit confusing.